### ISSUE TITLE
Wrong Image Naming Scheme for iPad @2x

### ISSUE THREAD

**Issue by winkelsdorf** [davbeck/TUSafariActivity#19]
Hi,

The images for iPad have wrong names. Wondering why nobody stumbled upon this earlier?

I created my own Activity for Chrome and searched why it's size was different compared to TUSafariActivity when on iPad.

<img width="409" alt="screenshot 2015-09-21 20 33 05" src="https://cloud.githubusercontent.com/assets/1413291/10001026/012e783e-60a0-11e5-9ae0-389cb4babb07.png">

Both TUSafariActivity Images, safari~iPad.png and safari~iPad@2x.png have the correct size, 76x76px and 152x152px. The padding is also equal to my own Chrome Images (20px transparent Pixels around the Icon on @2x).

<img width="1172" alt="screenshot 2015-09-21 20 44 57" src="https://cloud.githubusercontent.com/assets/1413291/10001356/a5948458-60a1-11e5-98aa-c08a88aa725d.png">

To track it down I manipulated the iPad Image:
<img width="350" alt="screenshot 2015-09-21 20 36 26" src="https://cloud.githubusercontent.com/assets/1413291/10001114/76946e26-60a0-11e5-9b00-76bc23aa6862.png">

*Guess what: The iPad Image is never loaded. The Simulator and Device were wrong as in the initial Image ;)*

**tl;dr**
You're doing it wrong. iPad Images are not meant to be named by a Scheme of "~iPad@2x" but "@2x~iPad". See http://stackoverflow.com/a/9627825/844907. It's been a long time since I had to deal with such kind of a Bug. That was fun to track it down ^^

I'll create a PR for renaming the Assets.

Thanks for TUSafariActivity, beside this little glitch a great help!

Edit: By the way, in PR #14 it was done right.

Cheers,
Frederik

**Comment by winkelsdorf** [comment]
Fixed with merge https://github.com/davbeck/TUSafariActivity/commit/0a736965d01b13852d094dbaff616f836f663fe5
